src/test/java/com/atlassian/db/replica/it/example/aurora/replica/api/SequenceReplicaConsistency.java

```diff
@@ -26,21 +25,16 @@
     private final ConcurrentHashMap<String, SuppliedCache<Long>> multiReplicaLsnCache;
     private final ReplicaNode replicaNode;
 
-    public static final class Builder {
```

Comment:
Maintaining this test, while it can be used because of none Aurora infra, it's an additional cost, without benefits hopefully soon we will have it.

Reply:
I think it's work in progress

Reply:
Yeah, 🤞 🤞 🤞 🤞

Reply:
I'm totally for it 🚀